plugins/juno_pipeline.py

```diff
@@ -27,15 +28,7 @@ def preflight(self, *args, **kwargs) -> bool:
         """
         Check if the target directory exists
         """
-        # if not os.path.exists("/pipe"):
-        #     raise ValueError(
-        #         "The pipeline directory does not exist. Please ensure the pipeline is mounted to /pipe."
-        #     )
-
-        # if not os.path.exists("/apps"):
-        #     raise ValueError(
-        #         "The apps directory does not exist. Please ensure the apps are mounted to /apps."
-        #     )
+        print('testing preflight')
```

Comment:
Is this needed?

Reply:
Ah I'll clean this up thanks!

Reply:
I removed the functions that weren't being customized and am defaulting to the base class for these.